nestkernel/conn_builder.cpp

```diff
@@ -239,97 +240,6 @@ nest::ConnBuilder::~ConnBuilder()
     delete it->second;
 }
 
-inline void
-nest::ConnBuilder::register_parameters_requiring_skipping_(
-  ConnParameter& param )
-{
-  if ( param.is_array() )
-  {
-    parameters_requiring_skipping_.push_back( &param );
-  }
-}
-
-inline void
-nest::ConnBuilder::check_synapse_params_( std::string syn_name,
```

Comment:
This method should not be inlined, it is simply too large and the time required to execute it will certainly be signficant compared to call overhead.